assets/styles/components/_wysiwyg.scss

```diff
@@ -0,0 +1,363 @@
+    // Lists
+    // ==========================================================================
+    ul,
+    ol {
+        position: relative;
+        margin: size-clamp('md') 0;
+
+        >li {
+            padding-left: rem(30px);
+        }
+    }
+
+    ul {
+        & > li {
+            position: relative;
+
+            & + li {
+                margin-top: 0;
+            }
+
+            &::before {
+                content: "";
+                position: absolute;
+                left: 0;
+                top: 9px;
+                width: 8px;
+                height: 8px;
+                background-color: currentColor;
+                border-radius: 100%;
+            }
+        }
+    }
+
+    ol {
+        counter-reset: counter;
+
+        & > li {
+            counter-increment: counter;
+            position: relative;
+
+
+            & + li {
+                margin-top: 0;
+            }
+
+            &::before {
+                position: absolute;
+                left: 0;
+                content: counter(counter);
+            }
+        }
+    }
+
+    // ==========================================================================
+    // Link
+    // ==========================================================================
+    a {
+        display: inline;
+        text-decoration: underline;
+        text-decoration-thickness: 1px;
+        color: currentColor;
+        word-wrap: break-word;
+
+        &:hover {
+            text-decoration-thickness: 2px;
+        }
+    }
+
+    // ==========================================================================
+    // Image
+    // ==========================================================================
+    img {
+        width: 100%;
+        margin: size-clamp('lg') 0;
+    }
+
+    // ==========================================================================
+    // iFrame
+    // ==========================================================================
+    iframe {
+        width: 100%;
+        height: auto;
+        aspect-ratio: 16 / 9;
+        margin: size-clamp('lg') 0;
+    }
+
+    // ==========================================================================
+    // Table
+    // ==========================================================================
+    table {
+        width: 100%;
+        margin: size-clamp('lg') 0;
+
+        & + p,
+        & + ul,
+        & + ol {
+            margin-top: 1em;
+        }
+
+        & + h1,
+        & + h2,
+        & + h3,
+        & + h4,
+        & + h5,
+        & + h6 {
+            margin-top: 1em;
+        }
+    }
+
+    thead {
+
+    }
+
+    th {
+        vertical-align: top;
+        text-align: left;
+        padding: 20px 10px;
+        border-bottom: solid 1px currentColor;
+    }
+
+    tr {
+    }
+
+    td {
+        border-bottom: solid 1px currentColor;
+        padding: 10px;
+        vertical-align: top;
+    }
+
+    caption {
+        padding: 10px;
+        caption-side: bottom;
```

Comment:
> caption-side: bottom;

The default is to position the caption above the table, this is a better position, especially if the table is tall.